frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingSdkOptionCard/EmbeddingSdkOptionCard.tsx

```diff
@@ -1,8 +1,9 @@
 import { t } from "ttag";
 
 import { useSetting } from "metabase/common/hooks";
+import { Badge } from "metabase/home/components/EmbedHomepage/Badge";
```

Comment:
This is not perfect but i'm ok as long as we end up replacing all usages

Reply:
feels a bit off seeing we use the `Badge` component from `EmbedHomePage` inside the embedding settings page.

I'd rename the current `components/Badge` to something else since it doesn't look anything like a badge
![image](https://github.com/user-attachments/assets/efd6c4c4-6242-444a-9db9-f8b072e28f1b)

And move this `components/EmbedHomepage/Badge` to `components/Badge`

Reply:
We'll "soon" have a badge from metabase/ui, we were waiting for Alessio who was on PTO: https://metaboat.slack.com/archives/C057WD5L0JG/p1726586842672059.
Once that's get to master, we'll refactor the usages, I'll create an issue for that now

Reply:
Sounds good then. Thanks for the pointer.

Reply:
https://github.com/metabase/metabase/issues/48189